test/box/fix-memory-corruption.test.lua

```diff
@@ -0,0 +1,23 @@
+env = require('test_run')
+fiber = require('fiber')
+net_box = require('net.box')
+
+test_run = env.new()
+
+test_run:cmd('create server test_server with script="box/fix-memory-corruption-server.lua"')
```

Comment:
Why do you need two new processes to test it? Why couldn't you just connect to self?

Reply:
fixed